packages/fluentui/perf-test-northstar/project.json

```diff
@@ -5,6 +5,7 @@
   "implicitDependencies": [],
   "tags": ["react-northstar"],
   "targets": {
+    "test-perf": { "dependsOn": ["bundle"] },
```

Comment:
we changed `test-perf` target in this PR to not depend on any task by default - we override it where necessary ( projects that dont use TS path aliases )